src/Compiler/Microsoft.AspNetCore.Mvc.Razor.Extensions.Version1_X/test/TestFiles/IntegrationTests/CodeGenerationIntegrationTest/IncompleteDirectives_Runtime.codegen.cs

```diff
@@ -36,6 +36,14 @@ public class TestFiles_IntegrationTests_CodeGenerationIntegrationTest_Incomplete
 #line default
 #line hidden
          Member_test { get; private set; }
```

Comment:
Why is this `test` and not `__UniqueIdSuppressedForTesting__`?

Reply:
Wen have two places where we set it. A whole bunch of other places were using the other one, so it made the diff huge. I'll submit a follow up PR to make them consistent.